### PR TITLE
[Windows] Install Git using choco

### PR DESCRIPTION
**Pull request by dibir-magomedsaygitov**
# Description
In scope of this PR we are switching Git installation to use `choco` formula. See details in related issue.

## Check list
- [x] Related issue / work item is attached
- [ ] Tests are written (if applicable)
- [ ] Documentation is updated (if applicable)
- [ ] Changes are tested and related VM images are successfully generated

### CODE REVIEW

File: images/win/scripts/Installers/Install-Git.ps1 [hunk 1]
```diff
@@ -3,33 +3,18 @@
...
+    "/VERYSILENT", `
+    "/NORESTART", `
+    "/NOCANCEL", `
+    "/SP-", `
+    "/CLOSEAPPLICATIONS", `
+    "/RESTARTAPPLICATIONS", `
+    "/o:PathOption=CmdTools", `
+    "/o:BashTerminalOption=ConHost", `
+    "/o:EnableSymlinks=Enabled", `
+    "/COMPONENTS=gitlfs")
```
Comment (maxim-lobanov) [inline images/win/scripts/Installers/Install-Git.ps1:17]: Should we get rid of `--ignore-dependencies` flag for hub in this case?

Comment (gep13) [reply images/win/scripts/Installers/Install-Git.ps1:17]: @maxim-lobanov yes, the ignore-dependencies can now be safely removed, as the required dependencies will now be in place.

Comment (dibir-magomedsaygitov) [reply images/win/scripts/Installers/Install-Git.ps1:18]: Thank you, `--ignore-dependencies` is removed now

**Review by al-cheb — APPROVED** [review]

**Review by gep13 — APPROVED** [review]
LGTM!

**Review by AlenaSviridenko — APPROVED** [review]

**Review by maxim-lobanov — APPROVED** [review]

**Pull request merged**